feature-libs/checkout/base/components/guards/checkout.guard.ts

```diff
@@ -19,7 +19,15 @@ import { ExpressCheckoutService } from '../services/express-checkout.service';
   providedIn: 'root',
 })
 export class CheckoutGuard implements CanActivate {
-  private readonly firstStep$: Observable<UrlTree>;
+  private readonly firstStep$: Observable<UrlTree> =
```

Comment:
1. The output is not the same. Before it calls `this.checkoutStepService.getFirstCheckoutStepRoute()`, it read the first step ("delivery address") even it is disabled. Now, it uses `this.checkoutStepService.steps$`. 
2. https://stackoverflow.com/questions/51494083/initializing-variables-inline-during-declaration-vs-in-the-constructor-in-angula

The changes is in spec. (actually, the change is: `this.checkoutStepService.steps$` replace `this.checkoutStepService.getFirstCheckoutStepRoute()`)

Reply:
1. yup I did say that as well, but why I say they're the same is because isn't `getFirstCheckoutStepRoute` === steps[0].routeName === this.checkoutStepService.steps$(**emitted** - steps[0].routeName)
2. Yup, I understand, but in the end is just preference right? Just wanted to know why you changed it

Reply:
`steps[0].routeName` is not dynamically changed, but `steps$` is.